.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -54,7 +56,20 @@ build_and_install_qemu() {
 
         pushd "${GOPATH}/src/${QEMU_REPO}"
         git fetch
-        git checkout "$CURRENT_QEMU_VERSION"
+        # if extra patches exist
+        if [ -n "${CURRENT_QEMU_COMMIT}" ]; then
+            git checkout "$CURRENT_QEMU_PATCHES_BRANCH"
+            git checkout "$CURRENT_QEMU_COMMIT"
+            # Apply required patches
+            QEMU_PATCHES_PATH="${GOPATH}/src/${PACKAGING_REPO}/obs-packaging/qemu-aarch64/patches"
+            for patch in ${QEMU_PATCHES_PATH}/*.patch; do
+                echo "Applying patch: $patch"
+                git am -3 "$patch"
```

Comment:
please use `git apply` or `patch` to avoid issues like this

```
*** Please tell me who you are.
Run
git config --global user.email "you@example.com"
git config --global user.name "Your Name"
```

Reply:
I'm kind of confused How this error occurred last time. The patch is under the format of mailbox, FWIT, it's ok to use `git am`. ;)